scripts/generate_users_and_groups.py

```diff
@@ -0,0 +1,24 @@
+from kitsune.users.tests import GroupFactory, UserFactory
+
+GROUPS = {
+    "staff": 15,
+    "small_group": 10,
+    "medium_group": 15,
+    "large_group": 1000,
+    "xlarge_group": 5000,
+}
+
+
+def generate_sampledata(options):
+    """Generate user and group sample data."""
+    for group_name, user_count in GROUPS.items():
+        g = GroupFactory(name=group_name)
```

Comment:
This should be enough to create the group without the need to call the `save()` method

Reply:
GroupFactory.create(args)